### DISCUSSION TITLE
Can't send e-mail confirmation

### DISCUSSION THREAD

**Discussion by deniswalle96** [rathena/FluxCP#393] [Q&A] [answered]
I just can't get it to work. I've read dozens of threads but nothing helped. I do have a business email, how do I setup it in the application.php file? This is my configuration:

'MailerFromAddress'			=> 'email@domain.com',		// The e-mail address displayed in the From field.
'MailerFromName'			=> 'Server',			// The name displayed with the From e-mail address.

**Comment by reunite-ro** [comment]
https://www.youtube.com/watch?v=aJxANXqVAzw

This is my very very old tutorial, have tried follow this and see ? then check inbox/spam

**Comment by Akkarinage** [comment, marked as answer]
Depending on where your business mailbox is hosted, you may need to configure DKIM for authorised sending, or consider using a platform like SendGrid to handle this for you.

**Reply by deniswalle96** [reply]
SendGrid worked, thanks!
